kubectl/subcommand.go

```diff
@@ -243,3 +243,14 @@ func InspectSubcommandInfo(args []string) (*SubcommandInfo, bool) {
 
 	return ret, false
 }
+
+func (sci *SubcommandInfo) SupportsPager() bool {
+	switch sci.Subcommand {
+	case Get,
+		Describe,
+		Logs,
```

Comment:
When testing, it seems that when using `kubectl get pods --watch` or `kubectl logs --follow` it kind of breaks the pager.

So this code probably needs to represent that.

```suggestion
	case Get:
		return !sci.Watch
	case Logs:
		return !sci.Follow
	case Describe,
```

The `SubcommandInfo.Follow` doesn't exist, but it should be easy enough to add by looking at how `SubcommandInfo.Watch` is implemented. Basically just doing:

```diff
diff --git a/kubectl/subcommand.go b/kubectl/subcommand.go
index 345f65d..54dd058 100644
--- a/kubectl/subcommand.go
+++ b/kubectl/subcommand.go
@@ -10,6 +10,7 @@ type SubcommandInfo struct {
 	FormatOption FormatOption
 	NoHeader     bool
 	Watch        bool
+	Follow       bool
 	Help         bool
 	Recursive    bool
 	Client       bool
@@ -213,6 +214,8 @@ func CollectCommandlineOptions(args []string, info *SubcommandInfo) {
 			info.NoHeader = true
 		} else if args[i] == "-w" || args[i] == "--watch" {
 			info.Watch = true
+		} else if args[i] == "-f" || args[i] == "--follow" {
+			info.Follow = true
 		} else if args[i] == "--recursive=true" || args[i] == "--recursive" {
 			info.Recursive = true
 		} else if args[i] == "-h" || args[i] == "--help" {
```